reflex/components/component.py

```diff
@@ -471,6 +471,11 @@ def get_event_triggers(self) -> Dict[str, Any]:
             EventTriggers.ON_MOUNT: lambda: [],
             EventTriggers.ON_UNMOUNT: lambda: [],
         }
+        # Look at the fields for EventHandler types.
+        for field in self.get_fields().values():
+            if types._issubclass(field.type_, EventHandler):
+                default_triggers[field.name] = field.type_.args_spec
```

Comment:
```suggestion
                default_triggers[field.name] = getattr(field.type_, "args_spec", lambda: [])
```

In case someone just passes `on_click: rx.EventHandler`